packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -2527,7 +2527,9 @@ export interface IColorPickerProps {
     greenLabel?: string;
     hexLabel?: string;
     onChange?: (ev: React.SyntheticEvent<HTMLElement>, color: IColor) => void;
+    rectangleStyle?: IStyleFunctionOrObject<IColorPickerStyleProps, IColorPickerStyles>;

```

Comment:
should get removed if you run `update-api`

Reply:
done